### PR TITLE
Add ClickTeam Fusion

### PR DESCRIPTION
**Pull request by andOlga**
### SteamDB app page links to a few games using this
https://steamdb.info/app/251130/
https://steamdb.info/app/252010/
https://steamdb.info/app/319480/
https://steamdb.info/app/298630/
https://steamdb.info/app/248310/
https://steamdb.info/app/1151620/
https://steamdb.info/app/736260/
https://steamdb.info/app/601810/
https://steamdb.info/app/350070/
https://steamdb.info/app/274270/
https://steamdb.info/app/207530/
https://steamdb.info/app/315610/
https://steamdb.info/app/319510/
https://steamdb.info/app/332800/
https://steamdb.info/app/354140/
https://steamdb.info/app/506610/
https://steamdb.info/app/237740/
https://steamdb.info/app/777670/

### Brief explanation of the change
Addresses the second point of #74, detecting *some* Fusion games.
Unfortunately, this will have some false negatives, since the engine also supports exporting to a standalone .exe without any extra files to identify the game by, and that case cannot be addressed.

### CODE REVIEW

File: rules.ini [hunk 1]
```diff
@@ -43,6 +43,7 @@ ChromeEngine[] = (?:^|/)ChromeEngine\d\.dll$
 ChromeEngine[] = \.rpack$
 Clausewitz[] = (?:^|/)common/defines\.(?:lua|txt)$
 Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
+ClickTeamFusion = /mmfs2\.dll
```
Comment (xPaw) [inline rules.ini:46]: Use (?:^|/)

Comment (andOlga) [reply rules.ini:46]: The DLL is *always* in a subfolder. Should I make the change regardless?

Comment (xPaw) [reply rules.ini:46]: Yes because there's a little optimization we do for this.

Comment (andOlga) [reply rules.ini]: Done.

**Pull request merged**